### PR TITLE
[ch32786] Update attachments permissions

### PR DESCRIPTION
**Pull request by roman-karpovich**

### CODE REVIEW

File: src/unicef_attachments/fields.py [hunk 1]
```diff
@@ -47,17 +48,16 @@ def get_attachment(self, instance):
         return None
 
     def get_attribute(self, instance):
-        attachment = self.get_attachment(instance)
-        return attachment.file if attachment is not None else None
+        return self.get_attachment(instance)
```
Comment (robertavram) [inline src/unicef_attachments/fields.py:51]: will this have any consequences on backwards compatibility?

File: src/unicef_attachments/fields.py [hunk 2]
```diff
@@ -81,3 +81,37 @@ def __call__(self, serializer_field):
...
+
+    def to_representation(self, value):
+        if not value:
+            return value
+
+        request = self.context.get("request", None)
+        if request is None:
+            return value
+
+        return request.build_absolute_uri(value)
```
Comment (robertavram) [inline src/unicef_attachments/fields.py:103]: we're using blob storage, it's got some of its own magic. do we need this?

File: src/unicef_attachments/fields.py (hunk 1 above)
Comment (roman-karpovich) [reply src/unicef_attachments/fields.py:51]: rest this logic was moved to PermittedAttachmentField.get_attribute

File: src/unicef_attachments/fields.py (hunk 2 above)
Comment (roman-karpovich) [reply src/unicef_attachments/fields.py:103]: this logic doesn't depend on storage. here we replace relative backend url with absolute one
Attachment.file_link generates url which allows us to download attachment with respect of permissions, but it's not absolute

**Pull request merged**